client/components/Events/EventDateTime.tsx

```diff
@@ -72,15 +88,17 @@ export class EventDateTime extends React.PureComponent<IProps> {
                     date={start}
                     {...commonProps}
                 />
-                &ndash;
+                {hideDash && <>&ndash;</>}
```

Comment:
just wondering if it's called `hideDash` shouldn't the dash be rather removed? maybe this should be rather called `showDash`